### ISSUE TITLE
Changing cursor size?

### ISSUE THREAD

**Issue by jat255** [JezerM/web-greeter#56]
Thanks for this great project. Is it possible in the configuration file to change the size of the cursor? By default, my cursor is quite huge, and I'm not sure why. I see settings mentioned online for `lightdm-gtk-greeter`, but I'm not sure if there's similar customization functionality in `web-greeter`. Thanks!

**Comment by JezerM** [comment]
Nope, currently there's no setting for this. I will mark this as a feature request.
